### ISSUE TITLE
#1747043252------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1846]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-altitude" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-251d8a56-8747-6459-28cc-8ce82b0e7580">altitude</div>`

<strong>Element Location: </strong>#attr-label-altitude

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\1747043252_All page content should be contained by landmarks.png`
